flepimop/gempyor_pkg/tests/utils/test_random_distribution_sampler.py

```diff
@@ -0,0 +1,80 @@
+from functools import partial
+import inspect
+from typing import Any
+
+import numpy as np
+import pytest
+
+from gempyor.testing import partials_are_similar
+from gempyor.utils import random_distribution_sampler
+
+
+class TestRandomDistributionSampler:
```

Comment:
also feels a bit weirdly tautological. the special cases (where we have allowed parameter translations to the reference implementation) seems sensible.